RIPS/rip-7755.md

```diff
@@ -72,6 +72,12 @@ struct CrossChainCall {
   uint256 rewardAmount;
   // The minimum age of the L1 block used for the proof
   uint256 finalityDelaySeconds;
+  // An optional pre-check contract address on the destination chain
+  // Zero address represents no pre-check contract desired
+  // Used to verify if the Filler is authorized to perform the cross chain call
```

Comment:
Let's remove this or try to make more generic, "Can be used for arbitrary validation of fill conditions."